requirements/edx/base.txt

```diff
@@ -625,7 +625,7 @@ grpcio==1.65.4
     #   grpcio-status
 grpcio-status==1.62.3
     # via google-api-core
-gunicorn==22.0.0
```

Comment:
It has a few breaking changes, so we need to keep an eye on deployment.

> ** Breaking changes **
> 
> - refuse requests where the uri field is empty ([pull request 3255](https://github.com/benoitc/gunicorn/pull/3255))
> - refuse requests with invalid CR/LR/NUL in heade field values ([pull request 3253](https://github.com/benoitc/gunicorn/pull/3253))
> - remove temporary --tolerate-dangerous-framing switch from 22.0 ([pull request 3260](https://github.com/benoitc/gunicorn/pull/3260))
> - If any of the breaking changes affect you, be aware that now refused requests can post a security problem, especially so in setups involving request pipe-lining and/or proxies.

https://docs.gunicorn.org/en/stable/news.html#id1